README.md

```diff
@@ -81,6 +99,41 @@ request.then((response) => response.json())
 const fastRequest = timeoutFetch('https://example.com/fast', { timeout: 1e3 });
 ```
 
+### createRetryFetch [Server & Browser]
+```js
+import { createRetryFetch } from '@americanexpress/fetch-enhancers';
+```
+
+#### Configuring
+`createRetryFetch` accepts the an object with the following configuration: an *optional* maxRetry (default 3), and an *optional* back off strategy function:
+
+```js
+const enhancedRetryFetch = createRetryFetch()(fetch);
+```
+
+Optional retry count and back off strategy function that accepts the current retry count:
+```js
+const enhancedRetryFetch = createRetryFetch({
+  maxRetry: 5,
+  backoffStrategy: (n) => new Promise((res) => setTimeout(res, n * 1000)),
+})(fetch);
+```
+
+#### Example
+
+```js
+import { createRetryFetch } from '@americanexpress/fetch-enhancers';
+
+const retryFetch = createRetryFetch()(createTimeoutFetch(5e3)(fetch));
+
+// Then use retryFetch as you would normally
+const request = retryFetch('https://example.com');
+request.then((response) => response.json())
```

Comment:
might be more common to
```suggestion
retryFetch('https://example.com');
  .then((response) => response.json())
```
but not a blocker

Reply:
Agree. I was trying to keep it consistent with line #92